### PR TITLE
#26: Updated the documentation to make sure users of the plugin know …

### PR DESCRIPTION
**Pull request by vanlooverenkoen**
…why their interval is changed to 9000000

#26 should be better documented with this pr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,8 +69,10 @@ Workmanager.registerOneOffTask(
 // Periodic task registration
 Workmanager.registerPeriodicTask(
     "2", 
-    "simplePeriodicTask", 
-    frequency: Duration(hours: 1), //When no frequency is provided the default 15 minutes is set.
+    "simplePeriodicTask", \
```
Comment (timrijckaert) [inline README.md]: Maybe also duplicate this document in the `Workmanager.registerPeriodicTask` documentation

**Comment by timrijckaert** [comment]
Thanks Koen 🥳🎉

**Comment by vanlooverenkoen** [comment]
No problem. This is an awesome plugin.
I am currently testing it. I will let you know when I find some extra issues.

**Comment by timrijckaert** [comment]
Please do, we are still in a very experimental phase with this plugin.
We want to iron out the bugs + better iOS implementation first before calling it a `0.1.0` release.

We greatly appreciate the feedback and collaboration!

**Pull request merged**